src/common/config_opts.h

```diff
@@ -788,6 +788,7 @@ OPTION(osd_mon_heartbeat_interval, OPT_INT, 30)  // (seconds) how often to ping
 OPTION(osd_mon_report_interval_max, OPT_INT, 600)
 OPTION(osd_mon_report_interval_min, OPT_INT, 5)  // pg stats, failures, up_thru, boot.
 OPTION(osd_mon_report_max_in_flight, OPT_INT, 2)  // max updates in flight
+OPTION(osd_beacon_report_interval, OPT_INT, 500)       // (second) how often to send beacon message to monitor
```

Comment:
is it too large? mon_osd_report_timeout is 900, I think we could set to mon_osd_report_timeout/3 to allow at least 3 messages missing?